contracts/adapters/vesper/VesperAdapter.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.6;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../../interfaces/IUniswapV2Pair.sol";
+import "../../libraries/UniswapV2Library.sol";
+import "../../BaseAdapter.sol";
+import "../../IMasterVampire.sol";
+import "./IVesperController.sol";
+import "./IVesperPool.sol";
+import "./IPoolRewards.sol";
+
+contract VesperAdapter is BaseAdapter {
+    address constant MASTER_VAMPIRE = 0xD12d68Fd52b54908547ebC2Cd77Ec6EbbEfd3099;
```

Comment:
@mdzor  master vampire's address should be passed on the constructor.
Please refer other adapters from the latest master branch.